### DISCUSSION TITLE
Mergify needs the author permission to update the base branch of the pull request. 🤔

### DISCUSSION THREAD

**Discussion by jcs090218** [Mergifyio/mergify#5125] [Q&A]
I'm not sure what I can do about this, https://github.com/nix-community/NUR/pull/757/checks?check_run_id=30107895316.

I'm the owner of that forked repo.

**Comment by jd** [comment]
Did you follow https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork ?

**Comment by jcs090218** [comment]
Yes. There is no "Allow edits and access to secrets by maintainers" checkbox.

**Reply by jd** [reply]
I have no idea why you don't see this checkbox. Could you reach to GitHub support to know why?

**Reply by jcs090218** [reply]
Okay, I'll ask GitHub about it. Thanks for the help!
